pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -654,8 +654,4 @@ def _truncated_bptt_enabled(self) -> bool:
 
     def _truncated_bptt_steps(self) -> int:
```

Comment:
I think this function and the one above (`_truncated_bptt_enabled`) could be removed and the code inlined.